### PR TITLE
Updated README and scripts

### PR DESCRIPTION
**Pull request by NotJustSid**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,274 @@
...
-## Prerequisites
-### Windows
-- [Qt](https://www.qt.io/) (`> 5`)
-- [OpenSSL](https://indy.fulgan.com/SSL/Archive/Experimental/openssl-1.0.2o-x64-VC2017.zip) (`OpenSSL v1.0.2`)
- - Visual Studio 2017 or 2019
- - Some sort of bash
 
-### Linux
-To build on Linux you're going to need to have Qt installed which is done by installing `qt5-default` and `libqt5websockets5-dev`.
+Umiko is the bot used on [TheCherno's Discord Sever](https://discord.gg/K9rMPsu9pQ). This repository is what makes Umiko; it's the source code and it is completely open source!
```
Comment (Gaztin) [inline README.md]: Typo. Should say "Server"

File: README.md [hunk 2]
```diff
@@ -1,40 +1,274 @@
...
+- Reward you with some (very useful) currency because you typed and you were lucky enough.
+- Host events which tweak with Umiko's normal behavior to keep the fun going.
+  
+...and many more features still in works!
 
-  - On Linux you are going to need to build it with **qmake** (to get it just install `qt5-qmake`).
+## 🙋‍♂️ Contributing
 
-## Building
+So you want to contribute? Awesome! Let's get you going. For starters, know that Umiko uses the **[QDiscord API](https://github.com/Gaztin/QDiscord)** to interface with Discord. This is already in the repo as a submodule, so you don't need to worry much.
```
Comment (Gaztin) [inline README.md]: QDiscord is not an API. It's a library that interfaces with the *Discord API*!

File: README.md [hunk 3]
```diff
@@ -1,40 +1,274 @@
...
+|`Project Properties` (`Alt + Enter`) > `Configuration Properties` > `Debugging` > `Command Arguments`|
+
+## ❓ Help and FAQ
+
+If you weren't able to properly carry out the setup process, or if you just want to know more Umiko in general, considering checking these questions before opening an issue:
+
+<details>
+<summary>Windows: I think I didn't add the Qt version properly in `init.sh`</summary>
+
+If you're using Visual Studio, you need not worry; you can add those using `Qt VS Tools` (a Visual Studio extension). Simply go to `Qt Options` and then use the file explorer to add the required directories after clicking on `Add`.
```
Comment (Gaztin) [inline README.md]: Why are we instructing them to use Qt VS Tools when we have premake-qt set up? Kinda defeats the whole point, dunnit?

**Review by Gaztin — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (NotJustSid) [reply README.md]: Woops, well yeah I meant that. Fixing it rn!

File: README.md (hunk 1 above)
Comment (NotJustSid) [reply README.md]: Thanks for pointing it out!

**Review by Gaztin — APPROVED** [review]

**Pull request merged**